### PR TITLE
Port fixes from https://github.com/krottiers/terraform-azurerm-avm-pt…

### PR DESCRIPTION
**Pull request by jaredfholgate**
## Description

Bugfix for #44 

## Type of Change

- [ ] Non-module change (e.g. CI/CD, documentation, etc.)
- [ ] Azure Verified Module updates:
  - [x] Bugfix containing backwards compatible bug fixes, and I have NOT bumped the MAJOR or MINOR version in `locals.version.tf.json`:
    - [x] Someone has opened a bug report issue, and I have included "Closes #{bug_report_issue_number}" in the PR description.
    - [ ] The bug was found by the module author, and no one has opened an issue to report it yet.
  - [ ] Feature update backwards compatible feature updates, and I have bumped the MINOR version in `locals.version.tf.json`.
  - [ ] Breaking changes and I have bumped the MAJOR version in `locals.version.tf.json`.
  - [ ] Update to documentation

# Checklist

- [x] I'm sure there are no other open Pull Requests for the same update/change
- [x] My corresponding pipelines / checks run clean and green without any errors or warnings
- [x] I did run all  [pre-commit](https://azure.github.io/Azure-Verified-Modules/contributing/terraform/terraform-contribution-flow/#5-run-pre-commit-checks) checks

### CODE REVIEW

**Comment by ChrisSidebotham** [comment]
@khushal08 - The proposed change is not currently tested in the deployment validation of the testExamples due to the requirements of an existing `er_circuit` I believe.

File: SUPPORT.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 This project uses GitHub Issues to track bugs and feature requests. Please search the existing issues before filing new issues to avoid duplicates. For new issues, file your bug or feature request as a new issue.
 
-Issues can be created and searched through for existing [issues here](https://github.com/Azure/terraform-azurerm-avm-ptn-virtualwan/issues).
+Issues can be created and searched through for existing [issues here](https://github.com/terraform-azurerm-avm-ptn-virtualwan/issues).
```
Comment (segraef) [inline SUPPORT.md:9]: typo?

Comment (jaredfholgate) [reply SUPPORT.md:9]: Interesting, that was a grept update.

Comment (jaredfholgate) [reply SUPPORT.md]: My mistake, I was setting the env vars wrong for grept. They should be:

`$env:GITHUB_REPOSITORY="Azure/terraform-azurerm-avm-ptn-virtualwan"`
`$env:GITHUB_REPOSITORY_OWNER="Azure"`

I assumed that GITHUB_REPOSITORY was just the repository name...

**Review by khushal08 — APPROVED** [review]
LGTM

**Pull request merged**